### PR TITLE
Add tests

### LINKED ISSUE #2: Add tests (yeah it doesn't have any test)
Well yes it's a shame, but I'm going to get to it...

Would definitely appreciate any ideas about testing though!

### PR DESCRIPTION
**Pull request by thiamsantos**
Closes #2

### CODE REVIEW

**Comment by thiamsantos** [comment]
@edisonywh Could you please take a look? 

There are some scenarios that I was not able to reproduce/assert in the tests, like the `exit/1` calls.

File: test/test_helper.exs [hunk 1]
```diff
@@ -1 +1,22 @@
 ExUnit.start()
+
+defmodule Committee.TestHelpers do
```
Comment (edisonywh) [inline test/test_helper.exs:3]: Love these little helpers here 👍

File: test/committee/helpers_test.exs [hunk 2]
```diff
@@ -0,0 +1,50 @@
+defmodule Committee.HelpersTest do
+  use ExUnit.Case
+  import Committee.TestHelpers
+  alias Committee.Helpers
+
+  describe "staged_files/0" do
+    test "returns the staged files", context do
+      in_tmp(context.test, fn ->
```
Comment (edisonywh) [inline test/committee/helpers_test.exs:8]: what does `context.test` return?

**Review by edisonywh — APPROVED** [review]
Bravo @thiamsantos 👍 👍 👍, thanks for taking the effort to champion this!

I especially loved the test helpers that you used here, very clean and concise and your tests are all very self-contained + readable, had a lot to learn from you here :)

I see that your test failed due to the `1.7` requirement (which you've fixed in #23). I've merged that so I'll be able to merge this PR as soon as you rebase from `master`!

As for checking out the `exit` calls, do you mean for when a hook check returns a non-zero exit code? Have you tried [`catch_exit/1`](https://hexdocs.pm/ex_unit/ExUnit.Assertions.html#catch_exit/1)? Quick glance seems like that could work.

Either way, we can go ahead and merge this then we can tackle the non-zero exit next!

File: test/committee/helpers_test.exs (hunk 2 above)
Comment (thiamsantos) [reply test/committee/helpers_test.exs:8]: The message of the test. In this case it is `returns the staged files`.

**Comment by thiamsantos** [comment]
> I see that your test failed due to the 1.7 requirement (which you've fixed in #23). I've merged that so I'll be able to merge this PR as soon as you rebase from master!

Sure, I'll rebase it right away.

> As for checking out the exit calls, do you mean for when a hook check returns a non-zero exit code? Have you tried catch_exit/1? Quick glance seems like that could work.

I tried really quickly with it, doesn't seem to work, but I may missed something. Maybe we should revisit it later and try again.

**Comment by thiamsantos** [comment]
@edisonywh branch rebased.

**Comment by edisonywh** [comment]
I've went ahead and merged this PR, thanks so much for your contributino @thiamsantos!

💯💯💯💯💯

**Pull request merged**